### DISCUSSION TITLE
There doesn't seem to be a way to entirely remove an attribute

### DISCUSSION THREAD

**Discussion by jackwh** [downforcetech/fluidxml.php#17] [Questions]
If you write an attribute to a node, it's only possible to empty it out, but the attribute tag will always remain on the node.

There should be some kind of `removeAttribute` function I think - [DOMElement has this built-in](http://php.net/manual/en/domelement.removeattribute.php).

**Comment by jackwh** [comment]
Whoops, sorry! I'm wrong. It wasn't in the docs, but I discovered it was possible like so:

```
$this->xml->query("//MyDocument/MyElements/Element[@AttributeToRemove]")->each(function ($index, $domElement) {
    $domElement->removeAttribute('AttributeToRemove');
});
```

**Comment by daniele-orlando** [comment]
Yes, perfect solution. That's exactly the way to remove an attribute.

Fluidxml doesn't re-implement what DOM performs quite well.
